### PR TITLE
FT: Add using documentation

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: Using.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+Now, save each role's ARN value from the output of the commands above (the field
+`Arn`). These will be needed to setup the replication configuration on a bucket
+in a later step. In this example, the source role ARN is
+`arn:aws:iam::668546647514:role/SourceRoleForS3Replication` and the target role
+ARN is `arn:aws:iam::779657758625:role/TargetRoleForS3Replication`.
+
+### Create and attach policies
+
+Create the a S3-role-permissions-policy.json with the contents:
```
Comment (philipyoo) [inline Using.md]: ~~a~~

**Comment by rahulreddy** [comment]
You should also add one more line at the ending showing that if you do head-object on the target you will see object with ReplicationStatus REPLICA.

File: Using.md (hunk 1 above)
Comment (bennettbuchanan) [reply Using.md]: Thank you. Updated.

**Comment by rahulreddy** [comment]
@ironman-machine r+

**Comment by rahulreddy** [comment]
Merging manually since failures are cause by unrelated flaky tests.

**Pull request merged**